### DISCUSSION TITLE
Cannot reset FireflyIII admin password

### DISCUSSION THREAD

**Discussion by git-deeper** [firefly-iii/firefly-iii#5543] [Firefly III] [answered]
Can I dig into the FireflyIII configuration files or the database and reset the FireflyIII admin password?

I'm new to FireflyIII. I installed it via Softaculous on a webhost. Worked fine. When upgrading FirefoxIII I made some sort of error. Sorry, I don't have details :-(  Then I couldn't login at  http://MYDOMAIN.COM/firefly/public/login (using my real domain).

I used the "I forgot my password" link. Unfortunately, the reset email does not get delivered because, probably, I foolishly did not set up SMTP in FireflyIII earlier. So I'm locked out.

One option would be to blow away FireflyIII completely and start over. I didn't have too much data in it. I'd rather not nuke it, but it wouldn't be tragic. Any tips on how to remove FireflyIII completely without FireflyIII admin password? I can access the full file system and database. Then I'd reinstall it. Great software! Thanks!

**Comment by JC5** [comment, marked as answer]
You can still configure email I think. I'm not sure how softaculous works. 

https://docs.firefly-iii.org/firefly-iii/advanced-installation/email/

If you can't edit these variables you'll have to nuke it and start over.

**Reply by git-deeper** [reply]
I was  unsuccessful with these entries:
MAIL_MAILER=log
MAIL_HOST=mail.[mydomain].com
MAIL_PORT=465
MAIL_FROM=me@[mydomain].com
MAIL_USERNAME=me@[mydomain].com   [my email address with account on this host]
MAIL_PASSWORD=[password]   // That's not the real password!
MAIL_ENCRYPTION=ssl

For MAIL_FROM, I also tried the admin's email in FireflyIII, which is the email whose FireflyIII password I am trying to reset.
I plan to nuke the install. Any tips on how to prepare to start over fresh???  Thank you!

**Reply by JC5** [reply]
There was no message in the log files of Firefly III?

Unfortunately, when dealing with Softaculous, I'm not sure what the best way forward is.

**Reply by git-deeper** [reply]
@JC5 Thank you for your prompt replies! I am up and running again, this time with working email. It was easy to uninstall and reinstall FireFlyIII with Softaculous. Once I followed the example for the .env file settings for the MAIL_... parameters found [here](https://docs.firefly-iii.org/csv/usage/email/), all was well. I very much appreciate your support. A contribution will be coming your way assuming the program works well once I have it configured.
